src/server/Document/component.jsx

```diff
@@ -73,6 +74,15 @@ const Document = ({
           />
         )}
         {scriptsAllowed && scriptTags}
+        {scriptsAllowed && (
+          <script
+            type="text/javascript"
+            /* eslint-disable-next-line react/no-danger */
+            dangerouslySetInnerHTML={{
+              __html: `document.documentElement.classList.remove("no-js");`,
```

Comment:
Any thoughts about https://caniuse.com/#search=classList? I can't see anywhere else we're using it.

Reply:
According to that grid we should be all good using `classList.remove()`?